vendor/github.com/containerd/containerd/oci/spec_opts.go

```diff
@@ -955,6 +955,11 @@ func WithCapabilities(caps []string) SpecOpts {
 		s.Process.Capabilities.Bounding = caps
 		s.Process.Capabilities.Effective = caps
 		s.Process.Capabilities.Permitted = caps
+		if len(caps) == 0 {
+			s.Process.Capabilities.Inheritable = nil
+		} else if len(s.Process.Capabilities.Inheritable) > 0 {
+			filterCaps(&s.Process.Capabilities.Inheritable, caps)
+		}
```

Comment:
FWIW; not (yet) back porting this bump to v27.1; I need to get my head around some of the changes here around capabilities (I know they were a bit of a footgun in the past, and I wanted to double check those changes).